### ISSUE TITLE
Provide LaTeX math in Metanorma XML in addition to MathML (tip: use plurimath)

### ISSUE THREAD

**Issue by Intelligent2013** [metanorma/metanorma-ieee#118]
Source ticket: https://github.com/metanorma/mnconvert/issues/161

Math in the source IEEE XMLs encoded in the LaTeX format:
```xml
<tex-math notation="LaTeX">$2\sqrt{WH}$</tex-math>
```

Adoc;
```adoc
latexmath:[$2\sqrt{WH}$]
```

converts into metanorma xml:
```xml
<stem type="MathML">
	<math xmlns="http://www.w3.org/1998/Math/MathML">
		<mrow>
			<mn>2</mn>
			<msqrt>
				<mrow>
					<mi>W</mi>
					<mi>H</mi>
				</mrow>
			</msqrt>
		</mrow>
	</math>
</stem>
```

We need to convert Metanorma xml backward to IEEE XML format.
Could we add the original math LaTeX format into Metanorma xml like this:
```xml
<stem type="LaTeX">$2\sqrt{WH}$</stem>
```

or as xml comment _before_ `<stem type="MathML">`:
```xml
<!-- LaTeX: $2\sqrt{WH}$ -->
<stem type="MathML">
```
Thanks.

**Comment by ronaldtse** [comment]
MathML to LaTeX math conversion is supported by the new plurimath gem. We need to file a ticket there. Ping @suleman-uzair .

**Comment by ronaldtse** [comment]
I wonder if in Metanorma XML for IEEE we should provide alternative math formats. Or maybe not, because STS XML supports embedding of multiple math formats:

* LaTeX math: https://www.niso-sts.org/TagLibrary/niso-sts-TL-1-0-html/element/tex-math.html
* MathML: https://www.niso-sts.org/TagLibrary/niso-sts-TL-1-0-html/element/mml-math.html

I noticed that 1127 / master-1-standard contains tex-math.

Perhaps we don't need to generate tex-math instead just using MathML with the `<mml-math>` element.

**Comment by ronaldtse** [comment]
IEEE XML supports LaTeX math only -- even though the schema supports MathML, IEEE systems do not, so that is a moot point.

This means that mnconvert, which generates IEEE XML, needs to produce LaTeX math. mnconvert does not have the capability to convert MathML into LaTeX math. This capability needs to be done via plurimath, which means that the Metanorma XML needs to contain LaTeX math as well as the typical MathML.

**Comment by opoudjis** [comment]
https://github.com/plurimath/plurimath/issues/40 has been unactioned this past three months.

**Comment by opoudjis** [comment]
We have already done this (again at your request, @Intelligent2013) for converting asciimath to MathML, again as a comment. However, that is done in Presentation XML, after the fact, converting the MathML back to asciimath. I think we would rather avoid that in this case, and preserve the original LaTeX alongside the MathML.

Let me for now convert the MathML back to LaTeX, but I'll also look into multiple format math expressions in the Semantic XML.

**Comment by opoudjis** [comment]
Blocked by https://github.com/plurimath/plurimath/issues/54

**Comment by opoudjis** [comment]
@Intelligent2013 There are issues with plurimath, but by intention now is: if LaTeX and asciimath expressions are to be output within `stem` for you, they will be output within the XML tags `latexmath` and `asciimath`, rather than using comments. So:

````
<stem>
  <math>... MathML ...</math>
  <asciimath> ... </asciimath>
  <latexmath> ... </latexmath>
</stem>
````

**Comment by opoudjis** [comment]
Need to parse MathML in standoc, to ensure it is valid.

**Comment by opoudjis** [comment]
Waiting on Plurimath fixes:

https://github.com/plurimath/plurimath/issues/65
https://github.com/plurimath/plurimath/issues/64

**Comment by opoudjis** [comment]
Plurimath clearly needs a *lot* of work; it needs to deal with anything we throw at it, and it simply doesn't at the moment. It is way premature to use it as a general parser for Metanorma.

**Comment by opoudjis** [comment]
Blocked by https://github.com/plurimath/plurimath/issues/90

**Comment by opoudjis** [comment]
Retest: https://github.com/plurimath/plurimath/pull/93/files

**Comment by opoudjis** [comment]
Also give @suleman-uzair details of how to run metanorma
